examples/dt/uni01alpha/README.md

```diff
@@ -0,0 +1,257 @@
+# Deployed Topology - Alpha
+
+This document contains a list of integration test suites that would be
+executed against the below specified topology of Red Hat OpenStack Services
+on OpenShift. It also contains a collection of custom resources (CRs) for
+deploying the test environment.
+
+## Purpose
+
+This topology is used for executing integration tests that evaluate the
+`default` backends of the below mentioned services.
+
+## Environment
+
+### Nodes
+
+| Role | Machine Type | Count |
+| ---- | ------------ | ----- |
+| Compact OpenShift | vm | 3 |
+| OpenStack Compute | vm | 2 |
+| Networker         | vm | 3 |
+| Test nodes        | vm | 2 |
+
+### Networks
+
+| Name | Type | Interface |
+| ---- | ---- | --------- |
+| Provisioning | untagged | nic1 |
+| Machine | untagged | nic2 |
+| RH OSP | trunk | nic3 |
+
+#### Networks in RH OSP
+
+| Name | Type |
+| ---- | ---- |
+| ctlplane | untagged |
+| internalapi | VLAN tagged |
+| Storage | VLAN tagged |
+| Tenant | VLAN tagged |
+
+### Services, enabled features and configurations
+
+| Service          | configuration    | Lock-in coverage?  |
+| ---------------- | ---------------- | ------------------ |
+| Cinder           | LVM/iSCSI/lioadm | Must have          |
+| Cinder Backup    | Swift            | Must have          |
+| Glance           | Swift            | Must have          |
+| Swift            | (default)        | Must have          |
+| Octavia          | (amphora)        | Must have          |
+| Horizon          | N/A              | Must have          |
+| Barbican         | (default)        | Must have          |
+| Telemetry        |                  | Must have          |
```

Comment:
Add here Ceilometer, Prometheus to the "configuration" section would be nice

Reply:
Thank you. 

Let me include it in the support services as the main focus here is `Telemetry`.